src/shims/linux_stubs.c

```diff
@@ -40,8 +40,9 @@ char* mach_error_string(mach_msg_return_t x) {
 void mach_vm_deallocate() { LINUX_PORT_ERROR();  }
 
 mach_port_t pthread_mach_thread_np(void) {
-  return (pid_t)syscall(SYS_gettid);
```

Comment:
this should not be touched, I just asked @dgrove-oss to remove this in another pull request and that will conflict